deploy/fb-submitter-chart/templates/cronjob_db_sweeper.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: fb-submitter-cron-db-sweeper-{{ .Values.environmentName }}
+  namespace: formbuilder-platform-{{ .Values.environmentName }}
+spec:
+  schedule: "0 * * * *"
+  successfulJobsHistoryLimit: 0
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: fb-submitter-api-{{ .Values.environmentName }}
+            image: "754256621582.dkr.ecr.eu-west-2.amazonaws.com/formbuilder/fb-submitter-api:{{ .Values.circleSha1 }}"
+            args:
+            - /bin/sh
+            - -c
+            - bundle exec rails runner 'DbSweeper.new.call'
```

Comment:
Interesting, never used `rails runner`.
Are there times when one is more suitable?  Or are they totally interchangeable?

Reply:
it's old. eg its mentioned here https://stackoverflow.com/questions/5053515/rails-runner-script-not-working asked over 8 years ago. The alternative is a rake task. I would occasionally use if i don't feel like I need to place it in a rake task